cves/CVE-2009-2901.yml

```diff
@@ -43,7 +43,14 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: |
+  By default, Tomcat automatically deploys any directories placed in a host's appBase.
+  This behaviour is controlled by the autoDeploy attribute of a host, which defaults to
+  true. After a failed undeploy, the remaining files will be deployed as a result of
+  the autodeployment process. Depending on circumstances, files normally protected by
+  one or more security constraints may be deployed without those security constraints,
+  making them accessible without authentication. This issue only affects Windows
+  platforms.
```

Comment:
For clarity, the program would attempt to deploy files, fail, then deploy the rest of the files, even after reporting a failure?

Reply:
Almost - in this case, the system would attempt to _un_deploy (clean up files), fail, and then deploy the rest of the files in an unsafe way, due to the autodeployment process. I'll update the description to better reflect this.